### ISSUE TITLE
sub categories of cond-mat 

### ISSUE THREAD

**Issue by laserroger** [everthemore/physics2vec#1]
Say, what if one want to extract information from cond-mat.quant-gas? 

My way is add subject reader fields ` 'subject':     ('textList', 'oai_dc:dc/dc:subject/text()')`

and add an `if record[1].getField('subject')[0] == subsection` before write into file, where subsection is an argument.

**Comment by everthemore** [comment]
Yes exactly, I think that's how I would've done it. I assume that those are
the correct keywords, but should be easy to check otherwise :)

**Comment by laserroger** [comment]
I checked and it works :) I will submit a pull request recently. 

BTW, your talk today at machine learning & physics conference is great.
